src/native/corehost/fxr/command_line.cpp

```diff
@@ -145,6 +145,7 @@ namespace
         if (mode == host_mode_t::apphost)
         {
             app_candidate = host_info.app_path;
+            // Use realpath since we want the path to the app through symlinks
             doesAppExist = bundle::info_t::is_single_file_bundle() || pal::realpath(&app_candidate);
```

Comment:
Maybe not in this PR, but it would be worth looking into this a bit higher level. For example in this case, the `host_info.app_path` might already be after symlink resolution (in fact it typically will be, since if you run `apphost.exe` we will populate this as resolved via the code in corehost which also uses realpath).

But there are other places which initialize the `host_info.app_path` to paths which are unresolved yet.

Would be nice to have a principle that it either is guaranteed resolved, or not (in which case the callers don't try to resolve at all).